mu4/scenes/palette/palettemodule.cpp

```diff
@@ -76,14 +105,47 @@ void PaletteModule::onInit()
 {
     using namespace Ms;
 
-    // create a score for internal use
-    gscore = new MasterScore();
-    gscore->setPaletteMode(true);
-    gscore->setMovements(new Movements());
-    gscore->setStyle(MScore::baseStyle());
-
-    gscore->style().set(Sid::MusicalTextFont, QString("Bravura Text"));
-    ScoreFont* scoreFont = ScoreFont::fontFactory("Bravura");
-    gscore->setScoreFont(scoreFont);
-    gscore->setNoteHeadWidth(scoreFont->width(SymId::noteheadBlack, gscore->spatium()) / SPATIUM20);
+    // init configuration
+    m_configuration->init();
+
+    // load workspace
+
+    auto workspaceManager = framework::ioc()->resolve<workspace::IWorkspaceManager>(moduleName());
```

Comment:
I think this logic deserves a separate class

Reply:
moved to separate class